### DISCUSSION TITLE
NavX Inversion

### DISCUSSION THREAD

**Discussion by willitcode** [BroncBotz3481/YAGSL-Example#158] [Q&A] [answered]
When using a NavX, does invertedIMU need to be set to true since the NavX is CW-positive? I remember seeing in the general troubleshooting discussion that this was the case, but I was wondering if the changes to YAGSL in 2024 meant that this was no longer true. Thanks!

**Comment by thenetworkgrinch** [comment, marked as answer]
There were changes that meant it should no longer be true. You should still check with the "When to invert" page of the docs to be sure.

https://yagsl.gitbook.io/yagsl

**Reply by willitcode** [reply]
Gotcha, thanks!
